### PR TITLE
BugFix: compute the db usage percentage on the amount of the partition instead of the remaining space (bsc#977888)

### PR DESCRIPTION
**Pull request by ncounter**
https://bugzilla.suse.com/show_bug.cgi?id=977888

### CODE REVIEW

File: src/smdba/postgresqlgate.py [hunk 1]
```diff
@@ -501,7 +501,7 @@ class Info:
             d_size_available = (info.available - d_size)
             overview.append((d_name, self._bt_to_mb(d_size),
                              self._bt_to_mb(d_size_available),
```
Comment (isbm) [inline src/smdba/postgresqlgate.py]: Then in this case please kill `d_size_available` variable and:
```python
...
    self._bt_to_mb(info.available - d_size),
...
```

Comment (ncounter) [inline src/smdba/postgresqlgate.py]: Yes, done.

**Comment by isbm** [comment]
:+1:

**Comment by ncounter** [comment]
Thank you.
Merged.

**Pull request merged**